build/README.md

```diff
@@ -19,18 +19,19 @@ and allows reading/writing files and resolving Dart source code.
 An interface into the dart [analyzer][pub:analyzer] to allow resolution of code
 that needs static analysis and/or code generation.
 
-## Differences between the `build` package and `pub`.
+## Differences between the `build` package and `barback`.
```

Comment:
[nit] keep a mention of pub in the header - this is how most of our users know it

Reply:
Done.